Dockerfile

```diff
@@ -7,17 +7,18 @@ ARG COMPILER_NIX_NAME="ghc961"
 ARG MINIMAL="true"
 ARG IOG="false"
 
-RUN apt-get update \
- && yes | apt-get install curl git jq nix zstd \
+RUN DEBIAN_FRONTEND=noninteractive \
+ && apt-get update \
+ && apt-get -y install curl git jq nix zstd \
  && curl -L https://raw.githubusercontent.com/input-output-hk/actions/latest/devx/support/fetch-docker.sh -o fetch-docker.sh \
  && chmod +x fetch-docker.sh \
  && SUFFIX='' \
  && if [ "$MINIMAL" = "true" ]; then SUFFIX="${SUFFIX}-minimal"; fi \
  && if [ "$IOG" = "true" ]; then SUFFIX="${SUFFIX}-iog"; fi \
  && ./fetch-docker.sh input-output-hk/devx $PLATFORM.$COMPILER_NIX_NAME$TARGET_PLATFORM${SUFFIX}-env | zstd -d | nix-store --import | tee store-paths.txt \
- && yes | apt-get remove curl jq nix zstd \
- && yes | apt-get autoremove \
- && yes | apt-get autoclean
+ && apt-get -y remove curl git jq nix zstd \
```

Comment:
I think this is fair to clean up the docker image as much as we can. We would always want to have all the tools we want to provide to the user in the shell. This is so that we know for sure what they are using.